### PR TITLE
feat: generate proof, week 96, 97

### PR DESCRIPTION
**Pull request by joshualyguessennd**
- week 96, 97 proof , generate script

### CODE REVIEW

File: ssp-balancer/scripts/tx.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "96": {
```
Comment (pajicf) [inline ssp-balancer/scripts/tx.json]: No need to persist this in a file imo, we can just post as a comment on the relevant PR. Wdyt?

**Comment by joshualyguessennd** [comment]
TX for week 96 =        
```
 "BAL": [
            "0xb1dd3761bbdab1cdbf99fd785bb67fc54396f51e728b37dd462b20a159f79a02",
            "0x2b2d0fe09e9cab8eb1a4736033d7bce74bbd988b3db97841fd4acb33ed36dfe2",
            "0x6930a958458fbe7e6ed6ca1c6aafab30a5ac1df35da34c6e56bf3fe0b60daacd"
        ],
        "TUSD": [
            "0x11713c522dc96ecf77cc1974c61cc203c4378b20d1eb84a8ff847dd68fd00fec",
            "0x1a45465015c4db13da7cd6c423438fab55d88dba1275699db1ee777d8bde91c1"
        ]
```

**Review by pajicf — APPROVED** [review]
Great work! 🚀

**Comment by joshualyguessennd** [comment]
Week 97
```
"TUSD": [
     "0x8c1a23f747e26b4d56c3729b85fc5a7f51fde943d8ecc87d3ed47b2d3bb2ba44",
     "0x8a60aef6e16c87804595e6ca3307998bcde97b25492cb0e66f5d6894f1fdce24"
]
```

**Pull request merged**